### ISSUE TITLE
DISCOVERY:  Che theme

### ISSUE THREAD

**Issue by catrobson** [fabric8io/fabric8-ux#426]
From Gorkem & Mario:

At minimum we can change the Che logos with OSIO ones, if you can point us to the logos to use.
But we can go further and create own default theme [1] [2].

I would appreciate if you can give it a thought and let us know how to proceed with this.

[1] https://eclipse.org/che/docs/assemblies/sdk-themes/index.html
[2] https://che.eclipse.org/hands-on-creating-custom-ide-themes-in-che-d1b0b2214423

Verification criteria:
* Assess current Che visual design compared with the rest of our branding and application framework
* Determine if we should create a custom theme for Che.  If so, work with Gorkem and Mario on creating the theme.
* Provide branded as needed with guidance for how to use the branding

**Comment by sdash-redhat** [comment]
As per discussion with Gorkem & team, the Openshift will have an upstream Che ide which will be at beginning closer to the hosted Che IDE at Codenvy. So accessed the code-envy and grabbed the HTML DOM with CSS/JS that represent each views that will help in inspecting the related CSS to hack into the theme to get the new theme. 

Get the saved HTMLs here ..just extract from zip and click to run - 
https://drive.google.com/open?id=0B5mldgz6Jdx5NE9Rdkt4V1lTYVE


Also attached some sceen-shots for quick view here: 
https://docs.google.com/presentation/d/1vb2BjyVOYHn4tBHSHg12lBjmmwDC4oiPCX_QfV74LvY/edit#slide=id.p4 

Hope this helps!
@catrobson @sunilmalagi @muruGanesan @mindreeper2420 @bdellasc @Essjaysee

**Comment by sdash-redhat** [comment]
Also Mario shared the following screenshots: 
https://docs.google.com/a/redhat.com/presentation/d/1ToX-COIgcZ7xBOARXHTZ-jfb9qG3bQjq_9ImZ_0UKbs/edit?usp=sharing

**Comment by mindreeper2420** [comment]
@sdash-redhat can you convert these to InVision screens instead of Google Slides? The slides are not publicly accessible.

**Comment by sdash-redhat** [comment]
@mindreeper2420 Thanks Adam for pointing out .  Here is Invision link as required: https://redhat.invisionapp.com/share/MJBUDP2T7
HTMLs are in assets folder.

**Comment by mindreeper2420** [comment]
@sdash-redhat there is also a Light Theme, as Che supports theming:

![screen shot 2017-05-23 at 3 56 21 pm](https://cloud.githubusercontent.com/assets/4032718/26373692/81fe9942-3fd0-11e7-868c-865e3fede979.png)

**Comment by catrobson** [comment]
@sunilmalagi is there anything left needed from the Dev team here or do you feel you handed off everything they were looking for already?

**Comment by sunilmalagi** [comment]
@catrobson Initially they wanted only Che branding to be replaced with our OSIO on 'About modal' and 'Landing' pages, I have provided the mockups to Gorkem and Mario and they are OK with it. 
Regarding Che theme aligning with OSIO, I did check with them couple of times asking about feasibility of the design, How far I can go and make the changes on the che mockups, so that they can incorporate all the changes I am going to propose, they are yet to revert me on the same.  I have forwarded you a mail for your reference.  From my end nothing is pending.
